### PR TITLE
sc97079 init basic upload action repo 

### PR DESCRIPTION
**Pull request by AdamTranquilla**
- init basic upload action repo

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  collectCoverageFrom: ['src/**/*.{ts,js}', '!<rootDir>/node_modules/', '!<rootDir>/build/'],
```
Comment (leah-is-offline) [inline jest.config.js:2]: might want to remove this line here

File: src/index.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import { context } from '@actions/github';
+
+import { getAllData, getData } from './get-data';
+import { makeComment } from './make-comment';
+import { sendDataComment } from './send-data';
+import { testMonoRepo } from './test-mono-repo';
+
+const run = async (): Promise<void> => {
+  try {
+    const monoRepo = getInput('monoRepo');
```
Comment (leah-is-offline) [inline src/index.ts:11]: remove this

File: src/index.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    const url = getInput('dependanotEndpoint');
+
+    if (monoRepo === 'true') {
```
Comment (leah-is-offline) [inline src/index.ts:14]: can remove this as well. were not handling mono repos via an input flag on coverage anyways

File: src/index.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+      try {
+        if (reportMessage === 'report') {
+          prData.message = await sendDataComment(url, prData);
+        }
+      } catch (error) {
+        console.log(`${error}, Could not send data, printing comment`);
+      }
+    }
+
+    console.log(`Repo ID: ${prData.repositoryId}`);
```
Comment (leah-is-offline) [inline src/index.ts:45]: are you wanting to log all this stuff in the build logs on the build workflow run right now ?

**Review by leah-is-offline — COMMENTED** [review]
requesting changes on collectionCoverageFrom , other comments are suggestions.

File: src/index.ts (hunk 4 above)
Comment (AdamTranquilla) [reply src/index.ts:45]: yeah wanted to see what info I can get out based on any pre-existing code

File: src/index.ts (hunk 3 above)
Comment (AdamTranquilla) [reply src/index.ts:14]: Oh whoops. Lost some fixes we made yesterday when I accidentally reset to master. This wasn't supposed to be here

**Pull request merged**